lib/travis/api/v3.rb

```diff
@@ -36,6 +37,8 @@ def location(env)
       InsufficientAccess  = ClientError        .create(status: 403)
       JobAlreadyRunning   = ClientError        .create('job already running, cannot restart', status: 409)
       JobNotCancelable    = ClientError        .create('job is not running, cannot cancel', status: 409)
+      JobUnfinished       = ClientError        .create('job for this log is not finished, please wait until job has finished before retrieving log', status: 409)
```

Comment:
thanks for catching that Sven, and for the suggestion. Have updated the
error message.

On Fri, Sep 2, 2016 at 3:18 PM, Sven Fuchs <notifications@github.com> wrote:

> In lib/travis/api/v3.rb
> <https://github.com/travis-ci/travis-api/pull/301#discussion_r77344146>:
>
> > @@ -36,6 +37,8 @@ def location(env)
> >        InsufficientAccess  = ClientError        .create(status: 403)
> >        JobAlreadyRunning   = ClientError        .create('job already running, cannot restart', status: 409)
> >        JobNotCancelable    = ClientError        .create('job is not running, cannot cancel', status: 409)
> > +      JobUnfinished       = ClientError        .create('job for this log is not finished, please wait until job has finished before retrieving log', status: 409)
>
> "retrieving"? should that be "removing"?
>
> how about
>
> job still running, cannot remove log yet
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/travis-ci/travis-api/pull/301/files/e9d838015f3320fa6f73db43f0d2fd4633943837#r77344146>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABrHX3TOu3aleewRGhuxt0aArjXkbn55ks5qmCIggaJpZM4JP5pu>
> .
>



-- 
Carla Drago
Software Developer
carla@travis-ci.com

Travis CI   Test and Deploy with Confidence.
www.travis-ci.com



Travis CI GmbH, Rigaer Str.8, 10247 Berlin, Germany | GF/CEO: Mathias
Meyer, Joshua Kalderimis | Contact: contact@travis-ci.org
Amtsgericht Charlottenburg, Berlin, HRB 140133 B | Umsatzsteuer-ID gemäß
§27 a Umsatzsteuergesetz: DE282002648